addon/mixins/validatable-input.js

```diff
@@ -165,7 +165,7 @@ export default Ember.Mixin.create({
     } else {
       parent.addClass('has-error');
       element.next('.input-error').remove();
-      element.after('<p class="input-error" role="alert">' + errorMessage + '</p>');
+      element.after(`<label class="input-error" role="alert" for="${element.attr("id")}">${errorMessage}</label>`);
```

Comment:
``label``s **should** be used for all form controls: ``select``, ``textarea``, and ``input`` elements.  The W3c spec will explain more.